docs/windows.md

```diff
@@ -72,21 +96,21 @@ kubectl apply -f https://github.com/antrea-io/antrea/releases/download/v1.6.0/an
 
 Then, you can run the following commands. [nssm](https://nssm.cc/) will install Antrea as a Windows service. Please ensure
 `nssm` is on your machine, which is a handy tool to manage services on Windows. NOTE: `<KubernetesVersion>`, `<KubeconfigPath>`
-`<KubeProxyKubeconfigPath>` and `<KubeletKubeconfigPath>` should be set by you. E.g.
+and `<KubeletKubeconfigPath>` should be set by you. `<KubeProxyKubeconfigPath>` is only for kube-proxy mode. E.g.
```

Comment:
you should update the link above (`kubectl apply -f https://github.com/antrea-io/antrea/releases/download/v1.6.0/antrea.yml`). I think you may want to make it `v1.13.0` even though it's not released yet, as this is the first version that will support proxyAll for Windows?

Reply:
and update `$TAG="v1.6.0"` below as well?

Reply:
Updated, thanks.